src/Microsoft.VisualStudio.ProjectSystem.Managed.VS/ProjectSystem/VS/ConnectionPoint/ConnectionPointContainer.cs

```diff
@@ -0,0 +1,42 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.Runtime.InteropServices;
+using Microsoft.VisualStudio.OLE.Interop;
+
+namespace Microsoft.VisualStudio.ProjectSystem.VS.ConnectionPoint
+{
+    /// <summary>
+    /// This implementation is a copy from CPS
+    /// </summary>
+    [ComVisible(true)]
+    public class ConnectionPointContainer : IConnectionPointContainer
```

Comment:
No, every dll in VS is not a public API, only those that are in the SDK and provide NuGet packages for them.